more/problem-sets-competitive-programming.md

```diff
@@ -84,6 +84,7 @@
 * [500 Data structures and algorithms interview questions and their solutions in C++](https://www.quora.com/q/techiedelight/500-Data-Structures-and-Algorithms-interview-questions-and-their-solutions)
 * [A2 Online Judge](https://a2oj.com/ps)
 * [Advent Of Code](http://adventofcode.com)
+* [AdventJS - 25 días de retos con JavaScript](https://adventjs.dev) - Miguel Ángel Durán «midudev» *(GitHub account requested, not required)*
```

Comment:
```suggestion
* [AdventJS - 25 días de retos con JavaScript](https://adventjs.dev) - Miguel Ángel Durán, midudev *(GitHub account requested, not required)*
```

"«midudev»" is a bit ugly, better to write it this way in my opinion

Reply:
midudev is his nickname

Reply:
I do not like the angular brackets so much, let's wait for another opinion

Reply:
If I understand this correctly I would just omit the attribution. Looks to me like midudev is an organizer not really an author.

Reply:
> If I understand this correctly I would just omit the attribution. Looks to me like midudev is an organizer not really an author.

Is the organizer and the developer at same time :wink: Also a GitHub Star: https://stars.github.com/profiles/midudev/